doc/tutorials/cf.md

```diff
@@ -276,6 +276,9 @@ alternating least squares update rules).  These include:
  - `NMFALSFactorizer`
  - `RegularizedSVD`
  - `QUIC_SVD`
+ - `BiasSVD`
+ - `SVDPlusPlus`
+ - `RandomizedSVD`
```

Comment:
I took a look and I think these names don't actually correspond to what's used in the code.  So instead of `RandomizedSVD`, we should have `RandomizedSVDPolicy`, etc.; I think there are errors in what was already there before too.